src/database/index.tsx

```diff
@@ -540,6 +540,22 @@ export const removeScenarioUser = async (
   }
 };
 
+const batchSetFacilityModelVersions = async (
+  modelVersions: ModelInputs[],
+  facilityDocRef: firebase.firestore.DocumentReference,
+  batch: firebase.firestore.WriteBatch,
+) => {
+  modelVersions.forEach((modelVersion: ModelInputs) => {
+    const modelVersionDoc = facilityDocRef
+      .collection(modelVersionCollectionId)
+      .doc();
+
+    batch.set(modelVersionDoc, modelVersion);
+  });
+
+  return batch;
```

Comment:
We end up mutating this `batch` object inside of this method so I don't know that there's any reason to return it.